README.md

```diff
@@ -377,3 +377,8 @@ This CVE has been reported in systems.text.regularexpressions.dll which is used
 * Delete any logs collected thus far and make sure that all copies are deleted. 
 * If you cannot upgrade for any reason, please ensure all debugging is disabled
 * If you are concerned about a potential compromise, contact Snowflake Customer Support for assistance with invalidating all active sessions/tokens. 
+
+3. Global http connection settings - 
+	Snowflake has identified an issue where the connector is globally enforcing TLS 1.2 and certificate revocation checks with connectors up to v1.2.1.
+	Starting with v2.0.0, the connector will set these locally. However note that connector is now targeting .Net framework 4.7.2. 
```

Comment:
Can you add a blank line before this so that this is a separate paragraph?

Currently, all of this is being treated as a single paragraph. ( https://github.com/snowflakedb/snowflake-connector-net/blob/04ce70ae0b67911c9c56ae693f0b84ceabfa2419/README.md#notice )